locales/bn/validation.php

```diff
@@ -11,9 +11,11 @@
 |
 */
 
+:attribute অবশ্যই গ্রহণ করতে হবে যখন :other সমান :value হয়
```

Comment:
I think you can delete this line ;)

Reply:
Are you sure I can delete this line ? `'accepted_if'          => 'এই ফিল্ডটি তখন গ্রগণ করা হবে যখন :other হবে :value',`

Reply:
That was a silly mistake, my bad.